connect/models/schemas.py

```diff
@@ -256,6 +257,17 @@ def make_object(self, data):
         return Marketplace(**data)
 
 
+class CountrySchema(BaseSchema):
+    name = fields.Str()
+    icon = fields.Str()
+    zone = fields.Str()
+
+    @post_load
+    def make_object(self, data):
+        from connect.models import Country
+        return Country(**data)
```

Comment:
Every model needs to know its schema for its deserialize method to work. Every schema needs to know its model for its make_object method to work. Thus, one of the files need to import the order locally to avoid a cyclic import.

Reply:
I understood you, all schemas in one file, therefore, such imports, it is okay for me